admin/database/database.go

```diff
@@ -111,6 +113,7 @@ type DB interface {
 	InsertOrganizationMemberUser(ctx context.Context, orgID, userID, roleID string) error
 	DeleteOrganizationMemberUser(ctx context.Context, orgID, userID string) error
 	UpdateOrganizationMemberUserRole(ctx context.Context, orgID, userID, roleID string) error
+	CountSingleUserOrganizationsForMemberUser(ctx context.Context, userID string) (int, error)
```

Comment:
Other places, it's `Singleuser` instead of `SingleUser`, so pick one (I think `Singleuser` looks cleanest to show separation)